src/core/reactor.cc

```diff
@@ -2685,6 +2685,8 @@ void reactor::register_metrics() {
             // total_operations value:DERIVE:0:U
             sm::make_counter("io_threaded_fallbacks", std::bind(&thread_pool::operation_count, _thread_pool.get()),
                     sm::description("Total number of io-threaded-fallbacks operations")),
+            sm::make_queue_length("io_sink_queue_length", [this] { return _io_sink.queue_length(); },
```

Comment:
note: I never quite understood what these more detailed metrics are, like `make_queue_length` or `make_total_bytes`. These map to counter or gauge or whatever, but they do store a bit of additional metadata based on the make function, but I didn't really understand how they are used.

Reply:
Yeah I don't think it's used anywhere right now.